docs/secondary-network.md

```diff
@@ -68,10 +68,13 @@ data:
 ```
 
 At the moment, Antrea supports only a single OVS bridge for secondary networks,
-and supports up to eight physical interfaces on the bridge. The physical
-interfaces cannot be the Node's management interface, otherwise the Node's
-management network connectivity can be broken after `antrea-agent` creates the
-OVS bridge and moves the management interface to the bridge.
+and supports up to eight physical interfaces on the bridge.
+
+Note: When you set the Node's primary NIC as the secondary bridge physical interface,
+if the Node IP is assigned via DHCP and the DNS is auto-configured, you may lose the
+DNS configs because of the interface state changes. Please consider to provide a static
```

Comment:
configs -> configuration

you may lose the DNS configuration after the interface is moved to the OVS bridge, because of the interface state change

Reply:
consider providing

Reply:
Updated.